src/step_definitions/api.js

```diff
@@ -0,0 +1,22 @@
+import { defineSupportCode } from 'cucumber';
+import config from '../core/config.helper';
+import RestApiService from '../rest/rest-api-service.js';
+
+const service = new RestApiService(config.apiUrl);
+
+defineSupportCode(function({ When, Then }) {
+  let fetchResult;
```

Comment:
I wonder, maybe we can store it in RestApiClass, as variable instead of fetchResults in step definitions @tgorskievo ? So we would be able to store it under, desired key in table (just an idea)

Reply:
No, it was discussed.